src/app/shared/model/feeder-episode.model.ts

```diff
@@ -69,4 +74,21 @@ export class FeederEpisodeModel extends BaseModel {
     }
   }
 
+  createLink(url: string): string {
```

Comment:
wonder if we should move that createLink stuff up to base model or out into a utility